keps/sig-api-machinery/2896-openapi-v3/README.md

```diff
@@ -489,31 +527,37 @@ will be enabled on some API servers and not others during the rollout.
 Similarly, consider large clusters and how enablement/disablement will rollout
 across nodes. -->
 
+Version skew is discussed in a section above during a rolling control plane upgrade.
+
 ###### What specific metrics should inform a rollback?
 
 <!-- What signals should users be paying attention to when the feature is young
 that might indicate a serious problem? -->
 
+Non 200 responses from the `openapi/v3` endpoint could indicate a problem. A long response time from the apiserver for OpenAPI requests could also be an indicator of a problem.
+
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
 <!-- Describe manual testing that was done and the outcomes. Longer term, we may
 want to require automated upgrade/rollback tests, but we are missing a bunch of
 machinery and tooling and can't do that now. -->
 
+n/a
+
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
 <!-- Even if applying deprecation policies, they may still surprise some users.
 -->
 
+No.
+
 ### Monitoring Requirements
 
 <!-- This section must be completed when targeting beta to a release. -->
 
 ###### How can an operator determine if the feature is in use by workloads?
 
-<!-- Ideally, this should be a metric. Operations against the Kubernetes API
-(e.g., checking if there are objects with field X set) may be a last resort.
-Avoid logs or events for this purpose. -->
+The OpenAPI path `/openapi/v3` is populated.
```

Comment:
The question is about the workloads. So an answer is to check e.g. audit logs. Do they include this endpoint?

Reply:
Or metrics, if we have a way to identify requests to that endpoint. I expect we don't.

Reply:
hmm, I don't believe so other than the apiserver logs. Should we expose something to indicate this path is being hit?

Reply:
Metrics would do, although it is limited as user-agents and IPs are not exposed. But it's probably good enough.

Reply:
Metrics are good enough.